docs/src/main/asciidoc/redis-dev-services.adoc

```diff
@@ -0,0 +1,39 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Dev Services for Redis
+
+:extension-status: preview
+include::./attributes.adoc[]
+
+=== DevServices (Configuration Free Redis)
```

Comment:
Any reason to not call this "Dev Services for Redis"? That's how similar guides are called for Kafka, AMQP or Apicurio Registry.

Reply:
@Ladicek Actually this is a result of refactoring, as it's already called `Dev Service for Redis` - as this is how I named it (to be in line with i.e. Kafka guide).

The line which you are referring to is a left-over, so I'll remove it as it doesn't seem to be needed:

![image](https://user-images.githubusercontent.com/476392/123759443-30dc5a80-d8c0-11eb-8875-2e08962dfb5d.png)